code/game/machinery/computer/dna_console.dm

```diff
@@ -1681,7 +1681,7 @@
   */
 /obj/machinery/computer/scan_consolenew/proc/on_scanner_close()
 	// Set the appropriate occupant now the scanner is closed
-	if(connected_scanner.occupant)
+	if(connected_scanner?.occupant)
```

Comment:
strange, on_scanner_close() is called by the scanner, so why is this ended up null to start with?

Reply:
At a guess, it's [because of this](https://github.com/tgstation/tgstation/blob/134a87d0e146ce54173913775fa9bf0529f954b3/code/game/machinery/computer/dna_console.dm#L229). Both `connected_scanner` and `linked_console` are created at same time, but this fails to check for scanner's existence and _remove_ `linked_console` from it.

Reply:
If that's the cause override set_operational to unlink it.